### PR TITLE
Feature/update react

### PR DESCRIPTION
**Pull request by aju**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -52,14 +53,17 @@
     "scss-unused-variables": "./scripts/find_scss_unused_variables.sh src/",
     "build": "gulp build",
     "start": "http-server ./dist -p 8000",
-    "prepush": "npm test",
+    "prepush": "yarn test",
```
Comment (matikrk) [inline package.json]: why you change this? There is no difference.
But if you wanna change do it also on line 60, task test

Comment (aju) [reply package.json]: why mix yarn and npm ?:D
I made changes as you requested

Comment (matikrk) [reply package.json]: everywhere yarn is ok 👍

File: src/components/content-box/pages/content-box.jsx [hunk 2]
```diff
@@ -78,7 +78,7 @@ const examplePart1 =
       </OverlayedBox>
       <SeparatorVertical />
       <Avatar size={AVATAR_SIZE.SMALL} imgSrc="https://source.unsplash.com/64x64/?dog" />
-      <Avatar size={AVATAR_SIZE.SMALL} iconType={ICON_TYPE.PROFILE} iconColor={ICON_COLOR.GRAY} />
```
Comment (kdzwinel) [inline src/components/content-box/pages/content-box.jsx]: what happened here?

**Pull request merged**